trashtalk/views/cleanups.py

```diff
@@ -177,25 +176,17 @@ def create():
     :return: redirect to cleanup/:id if successful, to cleanup/new if not
     """
     # TODO: Issue #9 - Prevent multiple accidental submissions (ex., when errors occur)
-    full_address= get_full_address(request.form.to_dict())
+    full_address = get_full_address(request.form.to_dict())
     try:
         geoloc = get_location(full_address)
-    except:
+    except:  # TODO: Issue 56 - Add custom exception CleanupException or something
         current_app.logger.exception("There was an error finding location.")
-        flash("s%: Address not found" % full_address)
+        flash("There was an error creating your cleanup. Please try again.")
         return redirect(url_for('cleanups.new'))
     else:
-        location=location_factory(geoloc._asdict())
+        location = location_factory(geoloc._asdict())
         cleanup_data = request.form.copy()
-        if not cleanup_data['image']:
-            current_app.logger.info("No image was found: replaced with default")
-            cleanup_data['image'] = "crossed_rakes.png"
-        else:
-            current_app.logger.info("Image found: ", cleanup_data['image'])
-        cleanup_data['start_time'] = request.form["start_time"]
-        cleanup_data['end_time'] = request.form["end_time"]
         cleanup_data['location'] = location
-        cleanup_data['host'] = current_user
```

Comment:
This sets the current host

Reply:
Yeah realized i saved it to a separate commit while working so it'll be back!